### ISSUE TITLE
Inline value provider show render case sensitivnes of term in editor

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#117374]
Testing #117288

Have a case insensitive `InlineValueVariableLookup`. Notice how the vscode inline variables are rendered respecting the casing of the casing provided by the inlineValueProvider.
I think it should respect the editor one, since that one is closest to the inline values and that way the user can more easily connect them

![Screenshot 2021-02-23 at 12 40 50](https://user-images.githubusercontent.com/1926584/108838884-8f24c400-75d4-11eb-9f85-7b3b71e6e41d.png)

**Comment by weinand** [comment]
I agree, the key (=variable) shown in the inline key/value pair should use the same casing as the variable in the corresponding source code line.

But this can be easily achieved by passing the **unmodified** variable into the `InlineValueVariableLookup`.
The case-insensitive lookup ensures that this variable name is matched against variables in the VARIABLES view that have a different casing.

So this works without changes in VS Code:
<img width="1171" alt="2021-02-23_22-15-55" src="https://user-images.githubusercontent.com/1898161/108909481-e488c180-7624-11eb-91c4-033411579aae.png">

What I changed in mock-debug for testing this:

<img width="1155" alt="2021-02-23_22-19-15" src="https://user-images.githubusercontent.com/1898161/108909767-42b5a480-7625-11eb-86e1-9ef0988ab2c4.png">

- I just added the 'i' to the regular expression to make it match case-insensitive
- added a "false" argument to the `InlineValueVariableLookup` to make matches case-insensitive
